lib/engines/content_block_manager/app/models/content_block_manager/content_block/version.rb

```diff
@@ -14,6 +14,10 @@ def field_diffs
       def event_details
         self[:event_details] || {}
       end
+
+      def is_embedded_update?
+        event_details["updated_embedded_object_type"] && event_details["updated_embedded_object_name"]
```

Comment:
as we know the shape of this data - it's two string always with the same key right? - would there be anything wrong with just adding columns to the versions table? 
Just wondering as the naming of `event_details` atm feels a bit vague to me, and I miss the security of strong types... If we want to keep the blob could it be under `embedded_object_metadata` or something like that?

Reply:
Yeah, I wasn't 100% happy with this approach TBH. I think individual columns might be OK actually. Lemme give that a go